orc-rt/include/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(ORC_RT_HEADERS
+    orc-rt-c/orc-rt.h
+)
+
```

Comment:
There should also be a custom command that copies every header in `ORC_RT_HEADERS` to output directory.

Reply:
Hmm…what's the reason here?

Reply:
The invariant we maintain in LLVM is that the output build directory has the same layout as the installation directory so you can use the just built toolchain without running the install step.